### PR TITLE
Create github-sponsorship.md

### PR DESCRIPTION
**Pull request by teresa-novoa-zz**
Mattermost Github sponsorship

### CODE REVIEW

File: contributors/contributors/github-sponsorship.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Mattermost Github sponsorship
```
Comment (justinegeffen) [inline contributors/contributors/github-sponsorship.md]:
```suggestion
# Mattermost GitHub Sponsorship
```

File: contributors/contributors/github-sponsorship.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+At Mattermost, we would like to empower our key contributors and sponsor them for 1 year. This is not a financial reward, but mostly a recognition.
+With [GitHub Sponsors](https://docs.github.com/en/github/supporting-the-open-source-community-with-github-sponsors/about-github-sponsors#about-github-sponsors)  allowing the developer community to financially support the people and organizations who design, build, and maintain the open source projects they depend on, directly on GitHub, we can now launch this new Community Program.
```
Comment (justinegeffen) [inline contributors/contributors/github-sponsorship.md]:
```suggestion
With [GitHub Sponsors](https://docs.github.com/en/github/supporting-the-open-source-community-with-github-sponsors/about-github-sponsors#about-github-sponsors) allowing the developer community to financially support the people and organizations who design, build, and maintain the open source projects they depend on, directly on GitHub, we can now launch this new Community Program.
```

File: contributors/contributors/github-sponsorship.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+## Who is eligible?
```
Comment (justinegeffen) [inline contributors/contributors/github-sponsorship.md]:
```suggestion
## Who is Eligible?
```

File: contributors/contributors/github-sponsorship.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+_Mattermost Github sponsorship_ is always on the table for [MVP](https://developers.mattermost.com/contribute/mvp/) (“Most Valued Professional”) type people in any of the [available regions](https://github.com/sponsors) supported by Github.
```
Comment (justinegeffen) [inline contributors/contributors/github-sponsorship.md]:
```suggestion
_Mattermost Github sponsorship_ is always on the table for [MVP](https://developers.mattermost.com/contribute/mvp/) (“Most Valued Professional”) type people in any of the [available regions](https://github.com/sponsors) supported by GitHub.
```

File: contributors/contributors/github-sponsorship.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+## Process outline:
```
Comment (justinegeffen) [inline contributors/contributors/github-sponsorship.md]:
```suggestion
## Process Outline
```

File: contributors/contributors/github-sponsorship.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+1. MM staff member recommends someone for sponsorship and writes 1-2 sentences about why.
```
Comment (justinegeffen) [inline contributors/contributors/github-sponsorship.md]:
```suggestion
1. A Mattermost staff member recommends someone for sponsorship and writes a short motivation to support their recommendation.
```

File: contributors/contributors/github-sponsorship.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+2. [MLT](https://handbook.mattermost.com/company/about-mattermost/list-of-terms#mlt) exec approves
```
Comment (justinegeffen) [inline contributors/contributors/github-sponsorship.md]:
```suggestion
2. [MLT](https://handbook.mattermost.com/company/about-mattermost/list-of-terms#mlt) exec member approves the recommendation.
```

File: contributors/contributors/github-sponsorship.md [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+3. Request sent to finance team
```
Comment (justinegeffen) [inline contributors/contributors/github-sponsorship.md]:
```suggestion
3. The request is sent to the Finance team.
```

File: contributors/contributors/github-sponsorship.md [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+4. Email sent to potential sponsor telling them what to do.
```
Comment (justinegeffen) [inline contributors/contributors/github-sponsorship.md]:
```suggestion
4. An email is sent to the potential sponsor letting them know about the next steps in the process.
```

File: contributors/contributors/github-sponsorship.md [hunk 10]
```diff
@@ -0,0 +1,12 @@
...
+Note: Continued sponsorship will be evaluated quarterly.
```
Comment (justinegeffen) [inline contributors/contributors/github-sponsorship.md]:
```suggestion
**Note:** Continued sponsorship will be evaluated quarterly.
```

File: contributors/contributors/github-sponsorship.md (hunk 9 above)
Comment (justinegeffen) [reply contributors/contributors/github-sponsorship.md]: If "sponsor" is the person being sponsored then I recommend rewording to:
"An email is sent to the contributor letting them know about the sponsorship and the next steps in the process."

**Comment by justinegeffen** [comment]
Thanks, @teresa-novoa! This is great. I've made some formatting suggestions and have a comment regarding the process. One other suggestion is to add a line break between the header and the body text. :)

**Comment by jasonblais** [comment]
@teresa-novoa Thanks!

Question: Who will the Mattermost staff member write the recommendation to? (Edit: to their manager, or directly to ChrisO?)

**Comment by teresa-novoa-zz** [comment]
@jasonblais you will own this program (as per Corey´s suggestion), so it will be direct manager, then the Manager to Chris (if he is not that one), always cc´you.

**Comment by jasonblais** [comment]
@teresa-novoa Makes sense. Can we also add who owns steps 2 and 3? E.g. does the staff member making the recommendation ask for approval from the MLT member and send a request to the Finance team?

Else looks good!

PS: @justinegeffen we'll want to add this to the LHS navigation list as well.

**Comment by justinegeffen** [comment]
I've updated the LHS nav as part of this PR, thanks @jasonblais. :D

File: contributors/contributors/github-sponsorship.md [hunk 11]
```diff
@@ -0,0 +1,11 @@
+# Mattermost GitHub Sponsorship
+At Mattermost, we would like to empower our key contributors and sponsor them for 1 year. This is not a financial reward, but mostly a recognition.
+With [GitHub Sponsors](https://docs.github.com/en/github/supporting-the-open-source-community-with-github-sponsors/about-github-sponsors#about-github-sponsors) allowing the developer community to financially support the people and organizations who design, build, and maintain the open source projects they depend on, directly on GitHub, we can now launch this new Community Program.
+## Who is Eligible?
+_Mattermost Github sponsorship_ is always on the table for [MVP](https://developers.mattermost.com/contribute/mvp/) (“Most Valued Professional”) type people in any of the [available regions](https://github.com/sponsors) supported by GitHub.
+## Process Outline
+1. A Mattermost staff member recommends someone for sponsorship and writes a short motivation to support their recommendation.
```
Comment (teresa-novoa-zz) [inline contributors/contributors/github-sponsorship.md]: 1. A Mattermost staff member recommends someone for sponsorship and writes a short motivation to support their recommendation. This should be address to direct manager and keep Jason Blais in the loop.

File: contributors/contributors/github-sponsorship.md [hunk 12]
```diff
@@ -0,0 +1,11 @@
...
+2. [MLT](https://handbook.mattermost.com/company/about-mattermost/list-of-terms#mlt) exec member approves the recommendation.
```
Comment (teresa-novoa-zz) [inline contributors/contributors/github-sponsorship.md]: 2. [MLT](https://handbook.mattermost.com/company/about-mattermost/list-of-terms#mlt) exec member approves the recommendation and notifies Jason Blais.

File: contributors/contributors/github-sponsorship.md [hunk 13]
```diff
@@ -0,0 +1,11 @@
...
+3. The request is sent to the Finance team.
```
Comment (teresa-novoa-zz) [inline contributors/contributors/github-sponsorship.md]: 3. The request is sent to the Finance team by Jason Blais.

File: contributors/contributors/github-sponsorship.md [hunk 14]
```diff
@@ -0,0 +1,11 @@
...
+4. An email is sent to the potential sponsor letting them know about the next steps in the process.
```
Comment (teresa-novoa-zz) [inline contributors/contributors/github-sponsorship.md]: 4. Jason Balis reach the potential sponsor letting them know about the next steps in the process.

File: contributors/contributors/github-sponsorship.md (hunk 12 above)
Comment (justinegeffen) [inline contributors/contributors/github-sponsorship.md]:
```suggestion
2. [MLT](https://handbook.mattermost.com/company/about-mattermost/list-of-terms#mlt) exec member approves the recommendation and notifies Jason Blais (@jasonblais).
```

File: contributors/contributors/github-sponsorship.md (hunk 13 above)
Comment (justinegeffen) [inline contributors/contributors/github-sponsorship.md]:
```suggestion
3. The request is sent to the Finance team by Jason Blais.
```

File: contributors/contributors/github-sponsorship.md (hunk 14 above)
Comment (justinegeffen) [inline contributors/contributors/github-sponsorship.md]:
```suggestion
4. Jason Blais sends the potential sponsor an email letting them know about the next steps in the process.
```

File: contributors/contributors/github-sponsorship.md (hunk 11 above)
Comment (justinegeffen) [inline contributors/contributors/github-sponsorship.md]:
```suggestion
1. A Mattermost staff member recommends someone for sponsorship and writes a short motivation to support their recommendation. This should be addressed to their direct manager as well as Jason Blais.
```

File: contributors/contributors/github-sponsorship.md [hunk 15]
```diff
@@ -0,0 +1,11 @@
+# Mattermost GitHub Sponsorship
+At Mattermost, we would like to empower our key contributors and sponsor them for 1 year. This is not a financial reward, but mostly a recognition.
+With [GitHub Sponsors](https://docs.github.com/en/github/supporting-the-open-source-community-with-github-sponsors/about-github-sponsors#about-github-sponsors) allowing the developer community to financially support the people and organizations who design, build, and maintain the open source projects they depend on, directly on GitHub, we can now launch this new Community Program.
+## Who is Eligible?
+_Mattermost Github sponsorship_ is always on the table for [MVP](https://developers.mattermost.com/contribute/mvp/) (“Most Valued Professional”) type people in any of the [available regions](https://github.com/sponsors) supported by GitHub.
```
Comment (justinegeffen) [inline contributors/contributors/github-sponsorship.md]:
```suggestion
_Mattermost GitHub sponsorship_ is always on the table for [MVP](https://developers.mattermost.com/contribute/mvp/) (“Most Valued Professional”) type people in any of the [available regions](https://github.com/sponsors) supported by GitHub.
```

**Review by jasonblais — APPROVED** [review]
I made one really minor change to wording, hope that's okay! I just thought it's faster. Approving. Thanks Teresa for putting this together!

**Comment by teresa-novoa-zz** [comment]
Thank you Jason!

**Review by natalie-hub — APPROVED** [review]
Looks great!

**Pull request merged**